app/models/manageiq/providers/azure/cloud_manager/cloud_database.rb

```diff
@@ -1,2 +1,95 @@
 class ManageIQ::Providers::Azure::CloudManager::CloudDatabase < ::CloudDatabase
+  supports :create
+  supports :delete
+
+  require 'azure-armrest'
+
+  def self.params_for_create
+    {
+      :fields => [
+        {
+          :component => 'text-field',
+          :id        => 'name',
+          :name      => 'name',
+          :label     => _('Cloud Database Name'),
+        },
+        {
+          :component    => 'select',
+          :name         => 'database',
+          :id           => 'database',
+          :label        => _('Database Type'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :options      => ['MySQL', 'SQL', 'MariaDB', 'PostgreSQL'].map do |db|
+            {
+              :label => db,
+              :value => db
+            }
+          end,
+        },
+        {
+          :component => 'text-field',
+          :id        => 'server',
+          :name      => 'server',
+          :label     => _('Database Server Name'),
+        },
+        {
+          :component => 'text-field',
+          :id        => 'resource_group',
+          :name      => 'resource_group',
+          :label     => _('Resource Group'),
+        },
+      ],
+    }
+  end
+
+  def self.raw_create_cloud_database(ext_management_system, options)
+    case options[:database]
+    when 'SQL'
+      db_client = get_sql_db_client(ext_management_system)
+    when 'MySQL'
+      db_client = get_mysql_db_client(ext_management_system)
+    when 'PostgreSQL'
+      db_client = get_postgresql_db_client(ext_management_system)
+    when 'MariaDB'
+      db_client = get_mariadb_db_client(ext_management_system)
+    end
```

Comment:
Thinking we might want an
```ruby
else
  raise ArgumentError, etc...
```

If someone passes in an invalid option (e.g. post from the API without going through the UI)